to_do_backend/to_do/views.py

```diff
@@ -15,10 +17,21 @@ class CreateUserViewSet(viewsets.GenericViewSet, CreateModelMixin):
     serializer_class = UserSerializer
     permission_classes = [permissions.AllowAny]
 
+
 # class CreateUserViewSet(viewsets.ModelViewSet, CreateModelMixin):
 #     queryset = User.objects.all()
 #     serializer_class = UserSerializer
 #     http_method_names = ['post']
 #     permission_classes = [permissions.AllowAny]
 
+class ToDoViewSet(viewsets.ModelViewSet, RetrieveModelMixin):
+    queryset = User.objects.all()
+    serializer_class = ToDoSerializer
+    permission_classes = [permissions.IsAuthenticated]
+
+    def get_queryset(self):
+        user = self.request.user
+        return ToDo.objects.filter(user=user)
```

Comment:
pozovi:
```suggestion
        return super().get_queryset().filter(user=user)
```
Pa onda od parenta nasledis sve kada vec overridujes